scripts/get_eclwatchip.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
```

Comment:
Please delete this, as the functionality can be implemented within Terraform.

Reply:
> Please delete this, as the functionality can be implemented within Terraform.

I didn't delete this perl script because "resource.null" doesn't support output. So, I used the same method.